### PR TITLE
new advisories for

### PR DESCRIPTION
**Pull request by Peter-Prochaska**
User enumeration with error messages
Information disclosure in email field dialog at sharing
Flooding logfiles with a 1 Bit BMP File

### CODE REVIEW

File: advisories/server-list-part.php [hunk 1]
```diff
@@ -1,13 +1,17 @@
...
+<a href="/security/advisory?id=oc-sa-2017-003">Flooding logfiles with a 1 Bit BMP File</a><br>
 <br/><p>Version 9.1.2</p>
 <a href="/security/advisory?id=oc-sa-2016-017">SMB User Authentication Bypass</a><br>
 <a href="/security/advisory?id=oc-sa-2016-018">Stored XSS in CardDAV image export</a><br>
 <a href="/security/advisory?id=oc-sa-2016-019">Reflected XSS in Gallery application</a><br>
 <a href="/security/advisory?id=oc-sa-2016-020">Content-Spoofing in "files" app</a><br>
 <a href="/security/advisory?id=oc-sa-2016-021">Content-Spoofing in "dav" app</a><br>
 <br/><p>Version 9.0.7</p>
-<a href="/security/advisory?id=oc-sa-2016-022">Bypass received read-only share permissions using read-write reshare</a><br>
+<a href="/security/advisory?id=oc-sa-2017-001">User enumeration with error messages</a><br>
```
Comment (PVince81) [inline advisories/server-list-part.php]: should this say 9.0.8 which is the fixed version ?

any reason you removed "bypass received read-only share perms" ?

File: advisories/advisory-side.php [hunk 2]
```diff
@@ -1,5 +1,7 @@
 <br/><p>ownCloud server 9.1.3</p>
-<a href="/security/advisory?id=oc-sa-2016-022">Bypass received read-only share permissions using read-write reshare</a><br/>
```
Comment (PVince81) [inline advisories/advisory-side.php]: @Peter-Prochaska still removed ? Should it be moved to the previous version 9.1.2 maybe ?

Comment (PVince81) [inline advisories/advisory-side.php]: oh well... I'll assume this is intentional, I see it was added below

Comment (Peter-Prochaska) [inline advisories/advisory-side.php]: Yes, advisory-side.php is only for the latest version. And oc-sa-2016-022 only affects Versions < 9.1.2.

**Pull request merged**